app/grandchallenge/algorithms/templates/algorithms/job_list_row.html

```diff
@@ -66,11 +67,15 @@
 <split></split>
 
 {% if object.status == object.SUCCESS %}
-    <a class="badge badge-primary"
-       title="View algorithm result"
-       href="{% url 'workstations:workstation-session-create' slug=algorithm.workstation.slug %}?{% workstation_query algorithm_job=object config=algorithm.workstation_config %}">
+    <button class="btn badge badge-primary"
+            title="View algorithm result"
+            data-session-control
+            data-session-origin="https://us-east-1.{{ request.site.domain }}"
+            data-create-session-url='{% url 'workstations:workstation-session-create' slug=object.algorithm_image.algorithm.workstation.slug %}'
+            data-workstation-query='{% workstation_query algorithm_job=object config=object.algorithm_image.algorithm.workstation_config %}'
```

Comment:
The lookup for the slug and config has changed here compared to the LHS. Does this introduce new n+1 queries?

Reply:
It doesn't make a difference because we're prefetching the algorithm on the jobs in the view, but I will anyway change this back to the more direct lookup.